### ISSUE TITLE
Sharing a file on nextcloud with simpletask

### ISSUE THREAD

**Issue by hrv231** [mpcjanssen/simpletask-android#1130]
Hello,

My wife and I are using Simpletask for our Shopping List, after setting up webdav, one phone is able to edit the file while the other receives a message "Save todo file failed".
On my nextcloud I have that file shared with her username, this used to work on the previous version, I recently updated the version to the latest on both phones.

I also updated Nextcloud, so I don't know now where the problem is.

Is there something I could run on simpletask to share with you and see if the problem is on the app?

Thank you for the great work with the app!

**Comment by hrv231** [comment]
Alright, I was able to find out the debug button:

1.- Getting metadata for https://mynextcloudserver.com/remote.php/dav/files/hrv231/shoppinglist.txt

2.- File conflict remote

3.- System out: Authenticating for response: url=https://mynextcloudserver.com/remote.php/dav/files/hrv231/shoppinglist_conflict_blablabla.txt

So simpletask is creating those conflict files on the fly..
I'm lost here, and my folder now has several of this conflict files.

**Comment by hrv231** [comment]
The bad part is that it doesn't let me change to another file because it says:
"Not switching files when changes are pending"

**Comment by gerroon** [comment]
See my entry in this bug, it seems like something is not right with the latest version

https://github.com/mpcjanssen/simpletask-android/issues/1126

**Comment by mpcjanssen** [comment]
@hrv231 please uninstall and install the latest version 10.9.3. 

Additional question? Why are you using the webdav version and not the nextcloud version directly?

**Comment by hrv231** [comment]
> 
> 
> @hrv231 please uninstall and install the latest version 10.9.3.
> 
> Additional question? Why are you using the webdav version and not the nextcloud version directly?

Good point, for some reason I was using the webdav version because I didn't know how to do an initial setup with the nextcloud version.
It seems that the first time I tried to use the nextcloud version, it didn't work for me because I was using the entire url example (  https://mynextcloudserver.com/remote.php/dav/files/hrv231/ )

After reading your question, I decided to give it another try, and this time I only type my server address example ( https://mynextcloudserver.com ) , then the username and password and it worked!!

As a suggestion, I would like to see on the help website, examples with screenshots of how to connect to each different app, that way is going to be easier for new users.

Second suggestion is also related to the documentation, when I go to settings/features, I don't know or don't have a link to read for each one of those options, good one that caught my eye is the "Edit shared tasks", I would love to know what that option is about. Reading "Edit Simpletasked tasks prior to saving" I don't get the idea, don't know if it is exactly editing the file on the phone and then it will be saved on the server?

I would love to know if that option is the right option to check for a case like me, that I share the same file to another nextcloud user that uses the same Simpletask App.

Thank you again @mpcjanssen , this App is amazing and I'm glad that you don't use any Trackers.

**Comment by mpcjanssen** [comment]
@hrv231 documentation is always the thing that comes last (unfortunately). I have limited time and writing documentation is simply not what I enjoy doing most.

The edit task before saving shows a popup when you share some text (using the Android builtin `Share` functionality)  so you can edit it before it gets saved. (this is a little bit of a lie, if you cancel the dialog in any way the tasks are saved regardless). It doesn't really have anything to do with the multiple user scenario.

**Comment by smichel17** [comment]
> documentation is always the thing that comes last (unfortunately).

I'll take this opportunity to again share this lovely quote from the [bup readme](https://github.com/bup/bup#things-that-are-stupid-for-now-but-which-well-fix-later):

> Documentation is the sort of thing that will never be great unless someone from outside contributes it (since the developers can never remember which parts are hard to understand).
